string joint_name
uint8 axis_state
---
string result